src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractTests.4.4.0.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ServiceModel;
+using Infrastructure.Common;
+using WcfService;
+using Xunit;
+
+public static class MessageContractTests_4_4_0
+{
+    [WcfFact]
+    [OuterLoop]
+    public static void Message_With_MessageHeaders_RoundTrips()
+    {
+        BasicHttpBinding binding = null;
+        IWcfService_4_4_0 clientProxy = null;
+        RequestBankingData_4_4_0 requestData = null;
+        ChannelFactory<IWcfService_4_4_0> factory = null;
+
+        // *** SETUP *** \\
+        try
+        {
+            binding = new BasicHttpBinding();
+            factory = new ChannelFactory<IWcfService_4_4_0>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_4_4_0_Basic));
+            clientProxy = factory.CreateChannel();
+
+            requestData = new RequestBankingData_4_4_0();
+            requestData.accountName = "Michael Jordan";
+            requestData.transactionDate = DateTime.Now;
+            requestData.amount = 100.0M;
```

Comment:
No issue, I just wanted to comment that I found the decimal literal suffix funny in this context as an amount. It reads like it's saying 100 million!! :sunglasses: